### PR TITLE
[README] Updated to support new format file from 3.5.1-02

### PR DESCRIPTION
**Pull request by albundy83**
nexus-oss-feature-3.x.y-features.xml file has a different format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,9 +100,9 @@ If you are trying to use the R plugin permanently, it likely makes more sense to
 * If you are using OSS edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
 * If you are using PRO edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-pro-feature/3.x.y/nexus-pro-feature-3.x.y-features.xml`
    ```
-         <feature prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
+         <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
    +     <feature prerequisite="false" dependency="false">nexus-repository-r</feature>
```
Comment (DarthHater) [inline README.md]: Following this logic, there should be a version="1.0.0" for r here. Go ahead and add and I'll merge this in

**Comment by albundy83** [comment]
Should I add hardcoded version:
```<feature version="1.0.0" prerequisite="false" dependency="false">nexus-repository-r</feature>```
or something like:
```<feature version="plugin_version" prerequisite="false" dependency="false">nexus-repository-r</feature>```
But I'm afraid it could be confusing, until further update, version="1.0.0" could be enough ?

**Comment by DarthHater** [comment]
I would add 1.0.0, and we can update as new versions come out. Dealers choice @albundy83 :)

**Comment by albundy83** [comment]
Here the update, have a good day :)

**Pull request merged**